consensus/blake3pow/consensus.go

```diff
@@ -216,6 +216,9 @@ func (blake3pow *Blake3pow) VerifyUncles(chain consensus.ChainReader, block *typ
 		if ancestors[uncle.ParentHash()] == nil || (!workShare && (uncle.ParentHash() == block.ParentHash(nodeCtx))) {
 			return consensus.ErrDanglingUncle
 		}
+		if uncle.PrimaryCoinbase().IsInQiLedgerScope() && block.NumberU64(nodeCtx) < params.ControllerKickInBlock {
```

Comment:
technically this block value should be based on the prime number, because all the controller stuff moved to prime, just realized this

Reply:
or we need a separate number for the zone i think to do this

Reply:
Fixed